index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>The carlot quiz</title>
+
+<link rel="stylesheet" type="text/css" href="quiz.css">
+<link rel="stylesheet" type="text/css" href="/bower_components/bootstrap/dist/css/bootstrap.min.css">
+</head>
+
+<!-- Put a standard Bootstrap navbar element at the top of your page.
+Put only two elements in the navbar.
+A text input field.
+A button. -->
+
+<!--  When you start typing into the navbar's text input, the description of
+the currently selected car should be bound to what you are typing in and match
+it exactly.  -->
+
+
+
+<body>
+  <div class="container">
+
+    <nav class="navbar navbar-default">
+      <form class="navbar-form navbar-left">
+        <div class="form-group">
+          <input type="text" class="form-control user-input">
+        </div>
+        <button type="submit" class="btn btn-default">Submit</button>
```

Comment:
Right now, because of this form, whenever you click this button or hit enter in the input field the page reloads. (meaning you lose all the edits to the descriptions you've made) To prevent this, you can just change the form tag to a div. `<form class="navbar-form navbar-left">` becomes `<div class="navbar-form navbar-left">`. For what it's worth